### ISSUE TITLE
Crash: JsonElement.kt - kotlinx.serialization.json.JsonElementKt.error

### ISSUE THREAD

**Issue by mohit23x** [refiner-io/mobile-sdk-react-native#5]
#### Package
"refiner-react-native": "0.0.8"

#### Crash Details

JsonElement.kt line 237
kotlinx.serialization.json.JsonElementKt.error

```
Fatal Exception: java.lang.IllegalArgumentException
Element class kotlinx.serialization.json.JsonLiteral (Kotlin reflection is not available) is not a JsonObject
kotlinx.serialization.json.JsonElementKt.error (JsonElement.kt:237)
kotlinx.serialization.json.JsonElementKt.getJsonObject (JsonElement.kt:160)
io.refiner.Refiner.showSurvey (SourceFile:35)
io.refiner.Refiner.checkAvailableSurvey (SourceFile:2)
io.refiner.Refiner.access$checkAvailableSurvey (SourceFile:1)
io.refiner.Refiner$e$a.emit (SourceFile:2)
kotlinx.coroutines.flow.StateFlowImpl.collect (StateFlow.kt:404)
kotlinx.coroutines.flow.StateFlowImpl$collect$1.invokeSuspend (Unknown Source:15)
kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith (ContinuationImpl.kt:33)
kotlinx.coroutines.DispatchedTaskKt.resume (DispatchedTask.kt:235)
kotlinx.coroutines.DispatchedTaskKt.resumeUnconfined (DispatchedTask.kt:191)
kotlinx.coroutines.DispatchedTaskKt.dispatch (DispatchedTask.kt:162)
kotlinx.coroutines.CancellableContinuationImpl.dispatchResume (CancellableContinuationImpl.kt:406)
kotlinx.coroutines.CancellableContinuationImpl.resumeImpl (CancellableContinuationImpl.kt:440)
kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$default (CancellableContinuationImpl.kt:429)
kotlinx.coroutines.CancellableContinuationImpl.resumeWith (CancellableContinuationImpl.kt:337)
kotlinx.coroutines.flow.StateFlowSlot.makePending (StateFlow.kt:288)
kotlinx.coroutines.flow.StateFlowImpl.updateState (StateFlow.kt:355)
kotlinx.coroutines.flow.StateFlowImpl.setValue (StateFlow.kt:320)
io.refiner.ui.SurveyViewModel$h$a.emit (SourceFile:2)
kotlinx.coroutines.flow.internal.SafeCollectorKt$emitFun$1.invoke (SafeCollector.kt:15)
kotlinx.coroutines.flow.internal.SafeCollectorKt$emitFun$1.invoke (SafeCollector.kt:15)
kotlinx.coroutines.flow.internal.SafeCollector.emit (SafeCollector.kt:87)
kotlinx.coroutines.flow.internal.SafeCollector.emit (SafeCollector.kt:66)
io.refiner.shared.businessmodel.RefinerModel$trackScreen$1.invokeSuspend (RefinerModel.kt:114)
kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith (ContinuationImpl.kt:33)
kotlinx.coroutines.DispatchedTask.run (DispatchedTask.kt:106)
android.os.Handler.handleCallback (Handler.java:938)

com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1135)

```

#### Note

The issue is majorly happening on Android 12 and 11 devices.

**Comment by chezmo** [comment]
Thanks @mohit23x ! We are looking into this at the moment. If you can see any pattern (e.g. happens for a specific survey only), please let us know.

**Comment by chezmo** [comment]
The issue was caused by trying to parse custom CSS styles as an object. This has been fixed in version [1.1.1](https://github.com/refiner-io/mobile-sdk-react-native/releases/tag/1.1.1)
